samples/community/Sandeep-1507/template.json

```diff
@@ -1,7 +1,5 @@
 {
   "Globals": {
-    "window_width": 1189,
```

Comment:
Sorry if I'm unaware of the naming convention, but are we keeping the folder name according to contributors or moving to something generic/mainstream.

Reply:
Inside community folder, we're allowing credits to the ones who created the sample template json. This is to incentivise new OMR templates getting submitted and also get new users onboarded for issue https://github.com/Udayraj123/OMRChecker/issues/40